templates/contribute/index.md

```diff
@@ -50,21 +50,21 @@ It's polite to prefix the branch name with your username, so it's easier for us
 rather than from your own fork, as CI works better this way.)
 
 Typical workflow:
-* To get started, you'll need a local copy of mathlib.
-* At https://github.com/leanprover-community/mathlib, click "Fork" in the top right, to make your own fork of the repository.
-  Your fork is at https://github.com/USER/mathlib.
-  Alternatively, if you've asked for write access you can just use https://github.com/leanprover-community/mathlib.
+* To get started, you'll need a local copy of mathlib4.
+* At https://github.com/leanprover-community/mathlib4, click "Fork" in the top right, to make your own fork of the repository.
+  Your fork is at [https://github.com/USER/mathlib4](https://github.com/USER/mathlib4).
+  Alternatively, if you've asked for write access you can just use [https://github.com/leanprover-community/mathlib4](https://github.com/leanprover-community/mathlib4).
 * Now make a local clone of the repository.
   ```
-  git clone https://github.com/USER/mathlib.git
-  cd mathlib
+  git clone https://github.com/USER/mathlib4.git
+  cd mathlib4
   ```
 * The steps above only need to be done once (not once for each contribution).
-* Now, each time you want to work on a new change to mathlib, create a new branch:
+* Now, each time you want to work on a new change to mathlib4, create a new branch:
   ```
   git checkout -b my_new_branch   # This creates a new branch
   ```
-  It's also fine to simply clone https://github.com/leanprover-community/mathlib.git,
+  It's also fine to simply clone https://github.com/leanprover-community/mathlib4.git,
   but you won't be able to push changes unless you've asked for permission.
   An alternative at this step is to use `leanproject`.
```

Comment:
This is a possible out-dated reference.

Reply:
Any mention of `leanproject` is outdated since `leanproject` is Lean 3 only.

Reply:
Removed `leanproject` and mentioned `lake exe cache get`.